.github/workflows/codecov.yml

```diff
@@ -0,0 +1,31 @@
+name: Codecov Workflow
```

Comment:
Can you run these formatting change for yml files through the validator: https://www.yamllint.com/
Make sure this is valid to avoid failing workflows

Reply:
<img width="881" alt="image" src="https://github.com/binary-com/deriv-com/assets/129206554/0463fdd4-9536-4036-924c-191bf588585d">

Looks valid

Reply:
no amir just copy/paste it (make sure you disable format on save)

```
---
name: Codecov Workflow
on:
  pull_request:
    branches:
      - "**"
  push:
    branches:
      - master
jobs:
  build:
    name: Build
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - name: Use Node.js 18.x
        uses: actions/setup-node@v1
        with:
          node-version: 18.x
      - name: install and make test coverage
        run: |
          npm ci
          npm run jest:coverage
      - name: Upload coverage to Codecov
        uses: codecov/codecov-action@v3.1.1
        with:
          directory: ./coverage
          fail_ci_if_error: true
          files: ./coverage/lcov.info
          name: codecov-umbrella
          verbose: true

```